compiler/testData/codegen/bytecodeText/inlineClasses/noAssertionsForInlineClassesBasedOnNullableTypes.kt

```diff
@@ -1,4 +1,5 @@
 // !LANGUAGE: +InlineClasses
+// IGNORE_BACKEND: JVM_IR
```

Comment:
What happens here?

Reply:
There should be no assertion because an inline class is unboxed into a parameter of type `Any?`, but the inline class PR is only a draft right now.